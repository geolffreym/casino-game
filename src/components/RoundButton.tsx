import { Button } from '@material-ui/core'
import styled from 'styled-components'

const RoundButton = styled(Button)`
  border-radius: 50px !important;
  text-transform: none !important;
  font-weight: bold !important;
  font-size: 1rem !important;
`

export default RoundButton
