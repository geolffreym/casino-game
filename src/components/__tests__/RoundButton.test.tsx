import React from 'react'
import { shallow } from 'enzyme'
import RoundButton from '@components/RoundButton'
import 'jest-styled-components'

/* eslint-disable no-undef */
describe('<RoundButton />', () => {
  it('should render', () => {
    const roundButtonComponent = shallow(<RoundButton />)
    expect(roundButtonComponent).toMatchSnapshot()
  })

  it('should render with white border-radius', () => {
    const roundButtonComponent = shallow(<RoundButton />)
    expect(roundButtonComponent).toHaveStyleRule('border-radius', '50px !important')
  })
})
