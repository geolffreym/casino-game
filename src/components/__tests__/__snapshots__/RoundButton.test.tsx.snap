// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RoundButton /> should render 1`] = `ShallowWrapper {}`;
